src/components/LazyLog/index.tsx

```diff
@@ -1180,65 +1194,19 @@ export default class LazyLog extends Component<LazyLogProps, LazyLogState> {
                         iconFindPrevious={this.props.iconFindPrevious}
                     />
                 )}
-
-                {/*
-                 // @ts-ignore */}
-                <AutoSizer
-                    disableHeight={this.props.height !== "auto"}
-                    disableWidth={this.props.width !== "auto"}
+                <VList
+                    className={`react-lazylog ${styles.lazyLog} ${
+                        this.props.wrapLines ? styles.wrap : ""
+                    }`}
+                    style={{ height: 800 }}
                 >
-                    {
-                        // @ts-ignore
-                        ({ height, width }) => (
-                            <VariableSizeList
-                                ref={this.state.listRef}
-                                className={`react-lazylog ${styles.lazyLog}`}
-                                {...this.props}
-                                height={this.calculateListHeight(height)!}
-                                width={
-                                    this.props.width === "auto"
-                                        ? width
-                                        : this.props.width
-                                }
-                                itemSize={this.getItemSize}
-                                initialScrollOffset={this.state.scrollToIndex}
-                                itemCount={
-                                    rowCount === 0
-                                        ? rowCount
-                                        : rowCount +
-                                          (this.props.extraLines || 0)
-                                }
-                                onScroll={(options) => {
```

Comment:
It looks like you removed onScroll and the ability for users to attach to onScroll event?  Also what about ScrollFollowing?